prow/cmd/deck/static/common/common.ts

```diff
@@ -216,65 +215,19 @@ export function getCookieByName(name: string): string {
   return "";
 }
 
-export function createRerunProwJobIcon(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string, rerunCreatesJob: boolean, csrfToken: string): HTMLElement {
```

Comment:
this function is moved from common.ts to rerun.ts, which makes senes to me. However GitHub is not smart enough to diff between files, would you mind separate this PR into two commits, one just copy and the other is incremental change?

Reply:
This change is already reflected in `move rerun functionality to rerun.ts` commit